Sources/Recent.php

```diff
@@ -911,6 +912,7 @@ function UnreadTopics()
 				AND t.id_last_msg >= {int:min_message}
 				AND COALESCE(lt.id_msg, lmr.id_msg, 0) < t.id_last_msg' . ($modSettings['postmod_active'] ? '
 				AND ms.approved = {int:is_approved}' : '') . '
+				AND COALESCE(lt.unwatched, 0) != 1
```

Comment:
possible performance degradation,
someone with a big dataset/setup should test this coalesce approache.

Reply:
Why? Concerned that it won't use the index?

Reply:
this concerns didn't exists,
since the awnser is known -> it can't use a index.
the question is,
do it matter that no index can be used.

Reply:
There was never an index on this column to begin with.

Reply:
yea i know, but still someone had to test if the new filter had somekind of negativ impact,
in this cases where filter didn't happen before.